### ISSUE TITLE
Error in spacing between variables and resources when there is a service level connector.

### ISSUE THREAD

**Issue by maheeka** [ballerinalang/composer#1918]
Check with below sample : 
```
import ballerina.net.http;
service Ecommerce {
    http:ClientConnector productsService = create http:ClientConnector ("http://localhost:9090");
    resource productsInfo(message m) {
    } 
}
```
<img width="1137" alt="image" src="https://user-images.githubusercontent.com/1029806/27020926-83a610bc-4f62-11e7-81b9-46edb5006b65.png">

**Comment by roonyh** [comment]
The gap between the variables pane and the resource def is expected since the connector declaration head is drawn above the resource def. But is drawn below variables panel by #2219

![image](https://user-images.githubusercontent.com/3872221/27731652-9583cd86-5dab-11e7-9dd3-53308e631296.png)

**Comment by nadeeshaan** [comment]
Closing the issue since the fix is merged (#2219)
